php-fpm/Dockerfile

```diff
@@ -28,23 +27,42 @@ RUN apk add --update \
     php7-tokenizer \
     php7-xml \
     php7-zlib \
-    composer \
     libsodium \
     make \
-    curl
+    curl \
+    shadow \
+    tzdata \
+    sudo \
+    zsh \
+	git \
+	bash
 
+RUN cp /usr/share/zoneinfo/Europe/Paris /etc/localtime && echo "Europe/Paris" >  /etc/timezone && apk del tzdata
 RUN rm -rf /var/cache/apk/* && rm -rf /tmp/*
 
-ADD symfony.ini /etc/php7/php-fpm.d/
-ADD symfony.ini /etc/php7/cli/conf.d/
+RUN addgroup -g 1000 www-data && adduser -u 1000 -G www-data -D www-data -s zsh
+RUN groupmod -g 1000 www-data 
+RUN echo "%sudo	ALL=(ALL:ALL) ALL" >> /etc/sudoers
+RUN usermod -u 1000 -g www-data www-data && echo "www-data:www-data" | chpasswd && addgroup sudo && adduser www-data sudo 
```

Comment:
Nope nope nope ! I don't want to run my application with root privileges. `www-data` needs to stay unprivileged. If we needs to run something which needs root privileges, just use `docker-compose exec php` to run it. This application should never have to manipulates things that require root privileges.